run

```diff
@@ -1,22 +1,25 @@
 #/bin/bash
 
-set -e 
+set -x
 
 SERVICES=$*
-if [[ -z "$SERVICES" ]]; then
-  SERVICES="regalions" # developer regalions
-  echo "Not building any services"
+if [[ -n "$SERVICES" ]]; then
+  echo "building services: $SERVICES"
+  SERVICES="-b $SERVICES" # developer regalions
+  echo "building services: $SERVICES"
 fi
 
 if [[ -z "$HONEYCOMB_API_KEY" ]]; then
   echo "please set HONEYCOMB_API_KEY, it's required"
   exit 1
 fi
 
-echo "Logging in to ACR, because it's so annoying when we forget."
+# make sure we're logged in to the damn image repository
+# TODO: the list succeeds just fine if we're not logged in. f u chatgpt
+# the list check takes 1s; logging in takes like 6
 scripts/login-acr.sh
 
-set -x
+# set -x
 
 # maybe someday the container registry name will change, but this is what it is now
-skaffold run -d mainacra1e0ec0b.azurecr.io -b $SERVICES --port-forward=user -l skaffold.dev/run-id=static
+skaffold run -d mainacra1e0ec0b.azurecr.io $SERVICES --port-forward=user -l skaffold.dev/run-id=static
```

Comment:
Are you sure we don't need the `-b` part here?

Reply:
that's being tacked on above in the script in line 8 or it is empty.